### PR TITLE
Runner Drag + Queen Neuro Nerf

### PR DESCRIPTION
**Pull request by Surrealaser**
Per title.

### CODE REVIEW

**Comment by Rohesie** [comment]
Okay for a temporary fix. Ideally we want them able to pull monkeys and other xenos fast and humans slow.

That said, testing changes locally is important. The queen retains the `shift_spits` power here when she shouldn't, for example.

**Comment by Rohesie** [comment]
@ninjanomnom , this is a xeno nerf so you'll might want to review it, but I feel it brings a better balance to the game. We've tested it and xenos tend to win in drawn-out games as normal.

File: code/modules/mob/living/carbon/xenomorph/Castes/Queen.dm [hunk 1]
```diff
@@ -74,7 +74,7 @@
 		/datum/action/xeno_action/emit_pheromones,
 		/datum/action/xeno_action/activable/gut,
 		/datum/action/xeno_action/psychic_whisper,
-		/datum/action/xeno_action/shift_spits,
+		// /datum/action/xeno_action/shift_spits,
```
Comment (ninjanomnom) [inline code/modules/mob/living/carbon/xenomorph/Castes/Queen.dm]: Dont comment code to remove it, just remove it

File: code/modules/mob/living/carbon/xenomorph/Castes/Queen.dm [hunk 2]
```diff
@@ -506,7 +506,7 @@
 			/datum/action/xeno_action/emit_pheromones,
 			/datum/action/xeno_action/activable/gut,
 			/datum/action/xeno_action/psychic_whisper,
-			/datum/action/xeno_action/shift_spits,
+			// /datum/action/xeno_action/shift_spits,
```
Comment (ninjanomnom) [inline code/modules/mob/living/carbon/xenomorph/Castes/Queen.dm]: ^

**Pull request merged**